SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/EdgeSetGeometryAlgorithms.inl

```diff
@@ -503,14 +503,14 @@ sofa::type::vector< double > EdgeSetGeometryAlgorithms<DataTypes>::computeRest2P
 }
 
 template<class Vec>
-sofa::type::vector< double > compute_2points_barycoefs(const Vec& p, const Vec& a, const Vec& b)
+sofa::type::vector< SReal > compute_2points_barycoefs(const Vec& p, const Vec& a, const Vec& b)
```

Comment:
I think this function is not used

Reply:
I am trying to make this PR not breakable so we may delete this function (and others...) in an other PR